frontend/src/features/permits/components/permit-list/ActivePermitList.tsx

```diff
@@ -19,7 +19,9 @@ export const ActivePermitList = () => {
 
   return (
     <div className="table-container">
-      <InfoBcGovBanner description="Refunds and amendments can be requested over the phone by calling the Provincial Permit Centre at Toll-free: 1-800-559-9688. Please have your permit number ready." />
+      <InfoBcGovBanner
+        msg="Refunds and amendments can be requested over the phone by calling the Provincial Permit Centre at Toll-free: 1-800-559-9688. Please have your permit number ready." 
```

Comment:
if not a `const` for the message, maybe consider a `const` for the toll free number at least? We show it in a lot of places..